datagrid_gtk3/tests/utils/test_transformations.py

```diff
@@ -0,0 +1,38 @@
+"""Data transformation utilities test cases."""
+
+import unittest
+
+from datagrid_gtk3.utils.transformations import degree_decimal_str_transform
+
+
+class DegreeDecimalStrTransformTest(unittest.TestCase):
```

Comment:
Just to comment, the existing transformations tests are being done here: https://github.com/nowsecure/datagrid-gtk3/blob/master/datagrid_gtk3/tests/test_datagrid-gtk3.py#L337

Although I think it would be better for them to be on another module, like the one you created here. I remember that I kept them there because there were a lot of existing "transformations test" (the code we used before the transformations =P) in it, but I might have made a bad decision here =P